### PR TITLE
network: don't activate bond/team devices regardless of --activate (#1358795)

### PR DESCRIPTION
**Pull request by rvykydal**
Resolves: rhbz#1358795

Port from rhel7-branch.

### CODE REVIEW

File: pyanaconda/network.py [hunk 1]
```diff
@@ -1524,6 +1529,49 @@ def _update_vlan_interfacename_ksdata(devname, ndata):
     if devname != default_ks_vlan_interface_name(ndata.device, ndata.vlanid):
         ndata.interfacename = devname
 
+def update_slaves_onboot_value(devname, value):
+    """Update onboot value in ifcfg files of device slaves
+
+    :param devname: name of device
+    :type devname: str
+    :param value: value of onboot setting
+    :type value: bool
```
Comment (vpodzime) [inline pyanaconda/network.py]: These can be written as ``:param str devname:`` and ``:param bool value:`` saving two lines! :)

File: pyanaconda/network.py [hunk 2]
```diff
@@ -1524,6 +1529,49 @@ def _update_vlan_interfacename_ksdata(devname, ndata):
...
+        ifcfg_path = find_ifcfg_file_of_device(devname)
+        if not ifcfg_path:
+            log.debug("network: can't find ifcfg file of %s", devname)
+            return retval
+        ifcfg = IfcfgFile(ifcfg_path)
+        ifcfg.read()
+        uuid = ifcfg.get('UUID')
+
+    # Find and update ifcfg files of slaves
+    for filepath in _ifcfg_files(netscriptsDir):
```
Comment (vpodzime) [inline pyanaconda/network.py]: Where does ``netscriptsDir`` come from here? Is it a global constant in the file?

**Comment by vpodzime** [comment]
Looks good to me otherwise.

**Comment by rvykydal** [comment]
jenkins, test this please

**Comment by rvykydal** [comment]
FAIL: network_bond_test (tests.dracut_tests.parse-kickstart_test.ParseKickstartTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenkins/workspace/anaconda-master-PR/tests/dracut_tests/parse-kickstart_test.py", line 202, in network_bond_test
    self.assertEqual(ifcfg_lines[4], 'ONBOOT="yes"\n', ifcfg_lines)
AssertionError: 'ONBOOT="no"\n' != 'ONBOOT="yes"\n'
- ONBOOT="no"
?         ^^
+ ONBOOT="yes"
?         ^^^
 : ['# Generated by parse-kickstart\n', 'HWADDR=""\n', 'MASTER="fd48ea6e-2383-42eb-b0ae-2842f9fc5f5c"\n', 'NAME="eth0 slave 1"\n', 'ONBOOT="no"\n', 'TYPE="Ethernet"\n', 'UUID="1fa9d0c7-cd4e-409b-927a-f0e57e7bd3ad"\n']

**Comment by rvykydal** [comment]
jenkins, test this please

File: pyanaconda/network.py (hunk 2 above)
Comment (rvykydal) [inline pyanaconda/network.py]: Yes

File: pyanaconda/network.py (hunk 1 above)
Comment (rvykydal) [inline pyanaconda/network.py]: good to know, I'll change it while I am updating the PR anyway

**Comment by rvykydal** [comment]
jenkins, test this please

File: pyanaconda/network.py (hunk 2 above)
Comment (vpodzime) [inline pyanaconda/network.py]: Would be nice to change it to ``NETSCRIPTS_DIR`` in some future patch then. :)

**Pull request merged**